### ISSUE TITLE
False positives for SA1013 and SA1513 with newlines inside interpolations

### ISSUE THREAD

**Issue by jnm2** [DotNetAnalyzers/StyleCopAnalyzers#3898]
C# 11+ allows newlines inside interpolations (https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-11#newlines-in-string-interpolations).

With the following code snippet:

```cs
_ = $"abc {"some long C# expression here, after which I want to indent"
    } def";
```

SA1013 and SA1513 are both reported on `}`. They both misidentify this as a closing brace of a block statement.
